### ISSUE TITLE
jOOQ 3.11.10 prints its version as 3.11.9

### ISSUE THREAD

**Issue by kxbmap** [jOOQ/jOOQ#8427]
### Expected behavior and actual behavior:
Expected: Prints 3.11.10
Actual:
```
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@  @@        @@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@        @@@@@@@@@@
@@@@@@@@@@@@@@@@  @@  @@    @@@@@@@@@@
@@@@@@@@@@  @@@@  @@  @@    @@@@@@@@@@
@@@@@@@@@@        @@        @@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@        @@        @@@@@@@@@@
@@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
@@@@@@@@@@    @@  @@  @@@@  @@@@@@@@@@
@@@@@@@@@@        @@  @  @  @@@@@@@@@@
@@@@@@@@@@        @@        @@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@  @@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  Thank you for using jOOQ 3.11.9
```

### Steps to reproduce the problem (if possible, create an MCVE: https://github.com/jOOQ/jOOQ-mcve):
Use jOOQ 3.11.10

### Versions: 

- jOOQ: 3.11.10
- Java:
- Database (include vendor):
- OS:
- JDBC Driver (include name if inofficial driver):

**Comment by lukaseder** [comment]
Thanks a lot for your report. Indeed, there seems to have been a mistake when upgrading version numbers. This cannot be fixed for version 3.11.10, as we cannot re-release the version to Maven Central. 3.11.11 will be fine again.

I'll keep this issue open in 3.11.11 as a reminder to check up on this.

**Comment by lukaseder** [comment]
This also affects generated code if the `@Generated` annotation is included in generated code:

```java
@Generated(
    value = {
        "http://www.jooq.org",
        "jOOQ version:3.11.9" // Should be 3.11.10
    },
    comments = "This class is generated by jOOQ"
)
```
